src/runtime_stack_check.js

```diff
@@ -17,8 +17,8 @@ function writeStackCookie() {
   // The stack grow downwards towards _emscripten_stack_get_end.
   // We write cookies to the final two words in the stack and detect if they are
   // ever overwritten.
-  {{{ makeSetValue('max', 0, '0x2135467', 'i32' ) }}};
-  {{{ makeSetValue('max', 4, '0x89BACDFE', 'i32' ) }}};
```

Comment:
I think you can change these from `i32` to `u32` to get the result you want.

Reply:
@sbc100 Done, thanks!